rad/src/operators/string.rs

```diff
@@ -25,6 +34,30 @@ pub fn to_float(input: &RadonString) -> Result<RadonFloat, RadError> {
         .map_err(Into::into)
 }
 
+pub fn to_int(input: &RadonString) -> Result<RadonInteger, RadError> {
+    i128::from_str(&input.value())
+        .map(RadonInteger::from)
+        .map_err(Into::into)
+}
+
+pub fn to_bool(input: &RadonString) -> Result<RadonBoolean, RadError> {
+    bool::from_str(&input.value())
+        .map(RadonBoolean::from)
+        .map_err(Into::into)
+}
+
+pub fn length(input: &RadonString) -> RadonInteger {
+    RadonInteger::from(input.value().len() as i128)
```

Comment:
I think clippy prefers `i128::from(x)`, but `impl From<usize> for i128` does not exist. There is a `TryFrom` implementation but I'm not sure if we should handle errors or just unwrap.

```suggestion
    RadonInteger::from(i128::try_from(input.value().len()).unwrap())
```